server/protocols/webadmin/webadmin-cassandra/src/main/java/org/apache/james/webadmin/routes/SolveMailboxInconsistenciesRequestToTask.java

```diff
@@ -26,12 +26,21 @@
 import org.apache.james.webadmin.tasks.TaskFromRequestRegistry;
 import org.apache.james.webadmin.tasks.TaskRegistrationKey;
 
+import com.google.common.base.Preconditions;
+
 public class SolveMailboxInconsistenciesRequestToTask extends TaskFromRequestRegistry.TaskRegistration {
     private static final TaskRegistrationKey REGISTRATION_KEY = TaskRegistrationKey.of("SolveInconsistencies");
 
     @Inject
     public SolveMailboxInconsistenciesRequestToTask(SolveMailboxInconsistenciesService service) {
         super(REGISTRATION_KEY,
-            request -> new SolveMailboxInconsistenciesTask(service));
+            request -> {
+                Preconditions.checkArgument(request.headers("Confirmation").equalsIgnoreCase("true"),
```

Comment:
`Confirmation` is not scary enough and the log message should be specific to "we require the cluster to be offline"

Reply:
Ok for the log message.

By what would you replace `Confirmation` then? `ALLOW-DATARACE-UNPROOF-OPERATION` is better?